etc/config.xml

```diff
@@ -55,5 +55,77 @@
                 <stockprice>0 8 * * *</stockprice>
             </cron>
         </sinchimport>
+        <csp>
+            <mode>
+                <storefront>
+                    <report_only>1</report_only>
+                </storefront>
+                <admin>
+                    <report_only>1</report_only>
+                </admin>
+            </mode>
+            <policies>
+                <storefront>
+                    <objects>
+                        <policy_id>object-src</policy_id>
+                        <self>0</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </objects>
+                    <base>
+                        <policy_id>base-uri</policy_id>
+                        <self>0</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </base>
+                    <frame-ancestors>
+                        <policy_id>frame-ancestors</policy_id>
+                        <self>1</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </frame-ancestors>
+                    <images>
+                        <policy_id>img-src</policy_id>
+                        <values>
+                            <value id="media.sitc" type="host">https://media.stockinthechannel.com</value>
```

Comment:
The CSP config in config.xml is only for control of the "policy flags" (self, inline, eval, dynamic). The whitelisting of hosts in each of the rule types is configured by csp_whitelist.xml (from every installed module), and the import already whitelists media.stockinthechannel.com [here](https://github.com/StockChannel/SinchImport/blob/master/etc/csp_whitelist.xml#L7) (it has since v3.1.2)